### ISSUE TITLE
Styles doesn't work on a basic components

### ISSUE THREAD

**Issue by juan77** [bahmutov/cypress-vue-unit-test#377]
Thank you for taking time to open a new issue. Please answer a few questions to help us fix it faster. You can delete text that is irrelevant to the issue.

## Is this a bug report or a feature request? bug report

If this is a bug report, please provide as much info as possible

- version Cypress 4.12.1
- platform macOS 10.15.1
- expected behavior The background and other styles of the title should be applied.
- actual behavior The style is not applied.

First of all, graat idea!! good work!!!

I would like to test this framework for an small component library, and I'm learning how to configure it (I use extensively cypress on an small application, and works flawless).

1. installed vue cli and initialized new app, with CSS preprocessor and Stylus as the framework.

```
npm i -g @vue-cli
vue create app-ui
vue add cypress-experimiental
```

2. then on Greetings.vue added and to 

```
<style>
.title {
  background: red;
  font-weight: bold;
  font-family: sans-serif;
}
</style>
```

3. Stared the cypress

```
npm run test:components
```

Here is the template 

```
<template>
  <h1
    data-testid="title"
    class="title"
  >Hello world! This is your first component tested by Cypress</h1>
</template>
```

I also tried without luck

```
<style scoped>
<style lang="stylus">
```

Why cannot apply the styles, I'm missing something?

Thanks,
juan.-

**Comment by bahmutov** [comment]
Could you post your entire Greeting.vue and spec files, please?

**Comment by juan77** [comment]
Hi Gleb,

The Greetings.spec.js was untouched, and the Greetings.vue only added the class and the styles tags.

Here are the files as it.

Greetings.vue
```vue
<template>
  <h1
    data-testid="title"
    class="title"
  >Hello world! This is your first component tested by Cypress</h1>
</template>

<style>
.title {
  background: red;
  font-weight: bold;
  font-family: sans-serif;
}
</style>
```

```javascript
/// <reference types="cypress" />
import { mount } from 'cypress-vue-unit-test';
import Greetings from '../../src/components/Greetings.vue';

describe('Greetings', () => {
  it('Works awesomely', () => {
    // mount command comes from
    // https://github.com/bahmutov/cypress-vue-unit-test
    mount(Greetings);

    // now we can use any Cypress command to interact with the component
    // https://on.cypress.io/api
    cy.get('[data-testid=title]').should('exist');
  });
});
```

Hope I'm missing something.

Thanks in advance,
juan.-

**Comment by JessicaSachs** [comment]
Hey Juan, can you publish your repository? I can't reproduce this :-(

**Comment by JessicaSachs** [comment]
This works for me on the latest Cypress 5.1.0 + Vue CLI. Please make sure you're installing `stylus` and `stylus-loader` as devDependencies, too, as per the Vue CLI docs [here](https://cli.vuejs.org/guide/css.html#pre-processors)

**Comment by juan77** [comment]
Hi Jessica, thanks for your response,

I tried to do a twist, and started from scratch again, with the instructions verbatim, and now the styles work fine as expected.

Looking back, I think my mistake was creating a new project with vue cli, choosing the dependencies from there by selecting stylus and even cypress.

Now I have only tried the default dependencies (babel, eslint) as mentioned in the instructions, and then added the dependencies later.

Thank four your time.
Juan.-
